app/routes/goal_routes.py

```diff
@@ -0,0 +1,42 @@
+from .helper_functions import create_instance, get_all_instances, get_one_instance, get_one_instance, delete_instance, update_instance,  add_tasks_to_model
+from app.models.goal import Goal
+from flask import Blueprint
+
+
```

Comment:
Great job separating your goal_routes and task_rights into separate files as well!